chart/templates/deployment.yaml

```diff
@@ -44,13 +44,13 @@ spec:
           livenessProbe:
             httpGet:
               path: /health
-              port: 10999
+              port: 8080
```

Comment:
have you tried running this? Port 8080 is already used by the distributor and all containers in the same pod  listen on the same network namespace so I don't see how this could work...